recipes-openxt/tss/tpm2-tools_3.1.3.bb

```diff
@@ -4,23 +4,16 @@ SECTION = "tpm"
 
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=91b7c548d73ea16537799e8060cea819"
-DEPENDS = "tpm2-tss openssl curl autoconf-archive pkgconfig"
-SRC_URI = "git://github.com/01org/tpm2-tools.git;protocol=git;branch=master;name=tpm2-tools;destsuffix=tpm2-tools \
-    file://0001-tpm2_listpcrs-use-TPM2_GetCapability-to-determine-PC.patch \
-    file://0002-listpcrs-remove-one-redundant-call-to-tpm-get-cap.patch \
-    file://0003-listpcrs-fix-for-unsupported-disabled-alg-in-L.patch \
+DEPENDS = "tpm2-tss openssl curl autoconf-archive pkgconfig libgcrypt"
+SRC_URI = "git://github.com/01org/tpm2-tools.git;protocol=git;branch=master;tag=${PV};nobranch=1 \
```

Comment:
For some reason I never noticed that the recipe was missing SRCREV. Adding that in there and removing the nobranch option did the trick, otherwise it wouldn't build without nobranch.

Reply:
Correction - this only works for tpm2-tss. I don't know why, but OE can't find the SRCREV that I'm using in the master branch or in the tag (even though they both contain that commit). The only thing that works is supplying nobranch=1 with the correct SRCREV hash.

Reply:
@tsirakisn This is because the tag `3.1.3` is not in the master branch or the tpm2-tools repository, instead it was tagged in branch `3.X`. Since this recipe is built from a tag, you can remove the `branch=master` and `nobranch=1` statements entirely.

Reply:
Adding branch=3.X did the trick, good call.